scripts/sync-admin-users.rb

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+require 'json'
+require 'yaml'
+require 'net/https'
+
+require File.expand_path("../lib/mail_credentials_helper", __FILE__)
+require File.expand_path("../lib/uaa_sync_admin_users", __FILE__)
+
+def get_uaa_target(api_url)
```

Comment:
@saliceti I agree, however I don't think shipping with this is going to cause a huge problem and would rather not make such a fairly large change now.